DESCRIPTION

```diff
@@ -70,7 +70,8 @@ Imports:
     lda,
     parallel,
     text2vec, 
-    tibble
```

Comment:
and what's tibble added to do?

Reply:
this adds readr for reading/writing the editable python script, rather than tibble. tibble was added for the StudyWideBarPlot here: https://github.com/bimberlabinternal/CellMembrane/blob/898a1563f3e9156459a76d5394f3c6dad3d2e518/R/PseudoBulk.R#L303